src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs

```diff
@@ -34,10 +34,10 @@ protected override bool CanFind(IMethodSymbol symbol)
             CancellationToken cancellationToken)
         {
             var typeName = symbol.ContainingType.Name;
-            var documentsWithName = await FindDocumentsAsync(project, documents, cancellationToken, typeName).ConfigureAwait(false);
+            var documentsWithName = await FindDocumentsAsync(project, documents, SupportsGlobalSuppression(symbol), cancellationToken, typeName).ConfigureAwait(false);
```

Comment:
this is confusing.  isn't htat effectively the same as `findInGlobalSuppressions: true`?

Reply:
See next comment.